update.sh

```diff
@@ -286,8 +288,13 @@ function do_bake_push_target() {
         output+=", \"${product}:${g_server_version}_${g_container_release}\""
     fi
 
-    if [ "${g_latest_version}" = "${g_server_version}" ]; then
-        output+=", \"${product}:latest\""
+    output+=", \"${product}:${maj_min}\""
+
+    if [[ "${g_server_version}" =~ ^[0-9]+\.[0-9]+\.[0-9]+\.[0-9]+$ ]]; then
```

Comment:
Git the regex a name. Also in the others we seemed to prefer `[.]` instead of `\.`